Pyrado/pyrado/algorithms/episodic/predefined_lqr.py

```diff
@@ -43,7 +43,7 @@
 from pyrado.utils.tensor import insert_tensor_col
 
 
-class LQR(Algorithm):
+class LQR(Algorithm, ExposedSampler):
```

Comment:
Does LQR have a sampler? I dont think so

Reply:
Well there is this sampler being initialized:
 https://github.com/famura/SimuRLacra/blob/20b17ada305641bcd55dae5063c0b614d386c918/Pyrado/pyrado/algorithms/episodic/predefined_lqr.py#L87-L89

Reply:
Right, it is an evaluation only sampler. I forgot about that one.